tox.ini

```diff
@@ -1,18 +1,22 @@
 [tox]
-# python3.1 issues with 2to3 for dependencies install
-# python3.2 issues with pip/tox
-envlist = copying,cli,py26,py27,py33,py34
-
-[tox:jenkins]
-skip_missing_interpreters = True
+envlist = copying,cli,{py27,py35}-{lint,tests}
 
 [testenv]
+whitelist_externals =
+    cli:   {[testenv:cli]whitelist_externals}
 deps=
     -rtests_utils/test-requirements.txt
-    py26,py32: -rtests_utils/pylint-python-2.6_3.2.txt
-    py32: -rtests_utils/coverage-python-3.2.txt
 commands=
-    python setup.py nosetests
+    tests: {[testenv:tests]commands}
+    lint:  {[testenv:lint]commands}
+    cli:   {[testenv:cli]commands}
+
+[testenv:tests]
+commands=
+    pytest -v iotlabcli --doctest-modules --cov=iotlabcli --ignore=iotlabcli/integration
```

Comment:
iotlabcli/integration doesn't pass with pytest and was not processed by nose. Skipping it until one find a way to make it work is fine.